stories/feed-input/feed-input.stories.js

```diff
@@ -5,86 +5,91 @@ export default {
     }
 };
 
 
 export const notEmpty = () => `
     <div class="fd-feed-input" role="region" aria-label="Feed input with one row text">
         <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
-        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-2" aria-label="Feed" rows="1" placeholder="Post something here">Textarea with a text makes submit button active</textarea>
-        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-2" aria-label="Feed message" aria-required rows="1" placeholder="Post something here">Textarea with a text makes submit button active</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Send" title="Send" aria-disabled="false"></button>
     </div>
 `;
 
-notEmpty.storyName = 'With some text';
+notEmpty.storyName = 'Input with text';
 notEmpty.parameters = {
     docs: {
-        storyDescription: 'Example with a one row text'
+        iframeHeight: 150,
+        storyDescription: 'The input with text displays some text within the field, which activates the submit button.'
     }
 };
 
 export const inputGrowth = () => `
     <div class="fd-feed-input" role="region" aria-label="Feed input height growth">
         <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
-        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-3" aria-label="Feed" rows="3" placeholder="Post something here">With every line\ntextarea growths up\nuntil it reaches the max height</textarea>
-        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-3" aria-label="Feed" aria-required rows="3" placeholder="Post something here">With every line\ntext area grows upward\nuntil it reaches the max height</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Send" title="Send" aria-disabled="false"></button>
     </div>
 `;
 
-inputGrowth.storyName = 'Multiple lines';
+inputGrowth.storyName = 'Multiline';
 inputGrowth.parameters = {
     docs: {
-        storyDescription: 'Every new line extends textarea height until maximum height is reached'
+        iframeHeight: 180,
+        storyDescription: 'Every new line extends textarea height until maximum height is reached. The multiline feed input displays a text field with multiple lines of text. When a new line is added to the field, the text area\'s height extends until the maximum height is reached.'
```

Comment:
Careful! Harmless typo here **"text area \ 's"

Also, I really recommend that you separate _text_ from _area_ in the documentation, even though it's paired together in HTML.

Reply:
Hey @jessmetivier `textarea` should be one word as it's referring to the html element `<textarea>`. https://www.w3schools.com/tags/tag_textarea.asp

Reply:
Okay, sounds good! It just needs to be written in "code" language, like this <code>textarea</code> to differentiate it.